.github/workflows/main.yml

```diff
@@ -183,3 +183,126 @@ jobs:
       exec_command: bash ./scripts/ci/test_pytorch_blade.sh
   NGC-TORCH-1110:
     uses: ./.github/workflows/ngc_reusable.yml
+  AArch64-CPU-TF280:
```

Comment:
Nit: I prefer to write a reusable workflow like the `cpu_reusable.yml`

Reply:
Actually, I tried `aarch64_cpu_reusable.yml` before. However, I found that the github secrets seems not available  inside the reusable file.